### PR TITLE
New tool that converts SARIF file to txt suitable for Emacs

### PR DESCRIPTION
**Pull request by dkloper**
Introduce new 'emacs' sub-command (derived from html command) that converts SARIF to a text file. When opened in emacs, this file will bring a compilation mode, allowing easy navigation among code scan issues.

### CODE REVIEW

**Comment by dkloper** [comment]
> 

@microsoft-github-policy-service agree

**Comment by balgillo** [comment]
Thank you for your contribution!  We need README.md updated as well, but as it's been so long I'll get this merged and then update README in a separate PR.

File: sarif/cmdline/main.py [hunk 1]
```diff
@@ -371,6 +372,15 @@ def _html(args):
     html_op.generate_html(input_files, args.image, output, multiple_file_output)
     return _check(input_files, args.check)
 
+def _emacs(args):
+    input_files = loader.load_sarif_files(*args.files_or_dirs)
+    input_files.init_default_line_number_1()
+    _init_path_prefix_stripping(input_files, args, strip_by_default=True)
+    _init_blame_filtering(input_files, args)
+    (output, multiple_file_output) = _prepare_output(input_files, args.output, ".txt")
+    emacs_op.generate_compile(input_files, output)
```
Comment (balgillo) [inline sarif/cmdline/main.py:381]: Missing parameter here, `multiple_file_output` - fixed in https://github.com/microsoft/sarif-tools/pull/21

**Comment by balgillo** [comment]
Docs added and fixed a bug under https://github.com/microsoft/sarif-tools/pull/21

**Pull request merged**